pages/featured/index.ftd

```diff
@@ -7,10 +7,10 @@
 -- site-records.template list templates:
 
 -- site-records.template: 
-title: Blog Template
-preview-url: https://blog.fifthtry.site/
-preview-img: $assets.files.assets.templates.blog-temp.png
-slug: blog
```

Comment:
We are removing blog template?

Reply:
What component are you creating?

Reply:
In `blog.fifthtry.site` we only have blog components that we are using inside `simple-blog-template.fifthtry.site`. So `blog.fifthtry.site` is not a template for blog we have `simple-blog-template`.